pep_sphinx_extensions/pep_zero_generator/constants.py

```diff
@@ -45,4 +45,10 @@
 .. _specifications: https://packaging.python.org/en/latest/specifications/
 .. _PyPA specification update process: https://www.pypa.io/en/latest/specifications/#specification-update-process
 """,
+    "releases": """\
+A PEP is written to specify the release cycle for each minor release of Python.
+See the `dev guide`_ for more information.
```

Comment:
Maybe pedantic, but using the full name of the devguide
```suggestion
See the `developer's guide`_ for more information.
```